File: Ruby/testP3.rb
# encoding: utf-8
module Civitas
require_relative "vista_textual.rb"
require_relative "controlador.rb"
require_relative "civitas_juego.rb"
require_relative "dado.rb"
    class TestP3
      def self.main
      vista = Vista_textual.new
      juego = CivitasJuego.new("Alonso", "Javito","Rauliyo" ,"Alberto","Adri" ,"Ali")
      Dado.instance.setDebug(true)
      controlador = Controlador.new(juego, vista)
      controlador.juega
      end
      main
    end
end

File: civitas/TestP3.java


/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;
import java.util.ArrayList;
import GUI.*;
import juegoTexto.*;
/**
 *
 * @author nacho-c
 */
public class TestP3 {
    public static void main(String[] args) {
       VistaTextual vista = new VistaTextual();
       
       ArrayList<String> jugadores = new ArrayList<>();
       jugadores.add("Stelio");
       jugadores.add("Puidgemont");
       CivitasJuego juego = new CivitasJuego(jugadores);
       
       
       Dado.getInstance().setDebug(true);
       /*Controlador controlador = new Controlador(juego,vista);
       controlador.juega();*/
    }
    
}
